ctapipe/io/tableloader.py

```diff
@@ -101,7 +101,7 @@ def _empty_telescope_events_table():
     Create a new astropy table with correct column names and dtypes
     for telescope event based data.
     """
-    return Table(names=TELESCOPE_EVENT_KEYS, dtype=[np.int32, np.int64, np.int16])
```

Comment:
Is this change compatible with the data model?

Reply:
Yes, that's the point, right?

https://github.com/cta-observatory/ctapipe/blob/ed5fdcc75744c8eb8e5a516b25b540ae1a367abf/ctapipe/containers.py#L76-L78

Reply:
Indeed.  I had in mind some smaller dtypes.

Reply:
Yes, that's the point, right?
dtypes here:

https://github.com/cta-observatory/ctapipe/blob/ed5fdcc75744c8eb8e5a516b25b540ae1a367abf/ctapipe/containers.py#L76-L78